content/contribute/elasticsearch-develop.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "ElasticSearch notifier development"
+weight: 11
+---
+
+To develop BotKube with Elasticsearch notifier enabled, install it locally:
```

Comment:
```suggestion
To develop BotKube with Elasticsearch notifier, follow these steps:
```
or 
```suggestion
The easiest way to develop BotKube with Elasticsearch notifier enabled is to install Elasticsearch on your local Kubernetes cluster.
```

in general, 
- it will be good to maintain the "header/overview" and "steps" sections or  change it but in all docs.
- IMO it doesn't read nice one after another,  "(..)  install it locally:", "1. Install Elasticsearch"